hack/util.sh

```diff
@@ -149,7 +154,7 @@ function start_os_server {
 	${sudo} env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
 	 --master-config=${MASTER_CONFIG_DIR}/master-config.yaml \
 	 --node-config=${NODE_CONFIG_DIR}/node-config.yaml \
-	 --loglevel=4 \
+	 --loglevel=4 "${SERVER_ARGS}" \
```

Comment:
> @deads2k as a user I would expect specifying something explicitly on command line will override any config default

I disagree.  I think that as a user, I'd expect a config to be mutually exclusive with any flag that duplicates information in that config.

@liggitt tie breaker?